### PR TITLE
Completed prep-javascript-variables exercise.

### PR DESCRIPTION
**Pull request by John-Baik**
image

### CODE REVIEW

File: prep-javascript-variables/index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = "John Wesley Baik";
+      console.log('value of fullName:', fullName);
+    var numberOfPets = "2";
```
Comment (scbowler) [inline prep-javascript-variables/index.html]: Here we want to use an actual number not a string with a number in it. So we want to remove the quotes and just have the number.

File: prep-javascript-variables/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      console.log('value of numberOfPets:', numberOfPets);
+    var likesVegetables = Boolean(false);
```
Comment (scbowler) [inline prep-javascript-variables/index.html]: While this does work it is overkill and more work than necessary. You can just directly use `true` or `false`.

File: prep-javascript-variables/index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      console.log('value of likesVegetables:', likesVegetables);
+     var nothing = null;
+      console.log('value of nothing:', nothing);
+     var never =
```
Comment (scbowler) [inline prep-javascript-variables/index.html]: Here you don't want the `=` sign. Should just be `var never;`

**Review by scbowler — CHANGES_REQUESTED** [review]
Good start but a few updates needed. Let me know if you have any questions.

**Comment by John-Baik** [comment]
image

**Pull request closed without merging**